### PR TITLE
Update nav

### PR DESCRIPTION
**Pull request by nwinter**

### CODE REVIEW

File: app/locale/en.coffee [hunk 1]
```diff
@@ -82,8 +82,8 @@
     about: "About"
     contact: "Contact"
     twitter_follow: "Follow"
-    teachers: "Teachers"
```
Comment (phoenixeliot) [inline app/locale/en.coffee]: Leave this in, or update `base-flat.jade` not to use it.

Comment (nwinter) [inline app/locale/en.coffee]: I did update that.

File: app/styles/style-flat.sass [hunk 2]
```diff
@@ -158,7 +158,7 @@ body[lang='ru'], body[lang='uk'], body[lang='bg'], body[lang^='mk'], body[lang='
       padding: 30px 10px
       width: 100%
       
-    @media (max-width: $screen-lg-min)
+    @media (max-width: $screen-md-min)
```
Comment (phoenixeliot) [inline app/styles/style-flat.sass]: I'd just switch the above two instances of this to use `$grid-float-breakpoint` if they're meant to be kept in sync with that variable anyway.

File: app/templates/base-flat.jade [hunk 3]
```diff
@@ -35,28 +35,33 @@ mixin accountLinks
...
+                ul.nav.navbar-nav.hidden-md.hidden-lg
                   li.disabled
                     a
                       img.img-circle.img-circle-small.m-r-1(src=me.getPhotoURL() class=(me.isTeacher() ? 'border-navy' : ''))
                       span= me.displayName()
                   +accountLinks
 
                 ul.nav.navbar-nav
-                  li.dropdown.hidden-xs.hidden-sm.hidden-md
+                  li.dropdown.hidden-xs.hidden-sm
```
Comment (phoenixeliot) [inline app/templates/base-flat.jade]: dang, guess our bootstrap is too old for `.hidden-md-down` etc

**Pull request merged**